e2e/test/scenarios/embedding/sdk-iframe-embedding-setup/helpers/index.ts

```diff
@@ -66,17 +70,24 @@ export const navigateToEntitySelectionStep = ({
     cy.findByText("Exploration").click();
   }
 
-  cy.log("navigate to the entity selection step");
-
   // exploration template does not have the entity selection step
   if (experience !== "exploration") {
+    cy.log("navigate to the entity selection step");
+    getEmbedSidebar().within(() => {
+      cy.findByText("Next").click(); // Entity selection step
+    });
+  }
+
+  if (experience !== "exploration" && !options.skipResourceSelection) {
```

Comment:
This skips the resource picker steps when `skipResourceSelection` is disabled, or we are in `exploration` template which does not support entity selection.